docs/changes/coordinates/14763.feature.rst

```diff
@@ -0,0 +1,4 @@
+Support for ``_equatorial_radius`` and ``_flattening`` is added in
+``BaseGeodeticRepresentation``.
+A new class ``StandardGeodeticRepresentation`` is added, inheriting from
```

Comment:
Right now, at some level this is an implementation detail. On the other hand, if anyone ever subclasses `BaseRepresentation` themselves, code would no longer work - they now need to subclass something else.

I think best may be to use different names! See other in-line comments.